Shopify/ruby-lsp
# Show require and require_relative completions on quotes

### Motivation

Previously, we were showing require completions on `/` and require_relative completions on `./`. However, that's not very useful, we can show all options the moment the developer adds the quotes for the string.

### Implementation

- Added `"` as a trigger character, so that we can match empty require arguments
- Adapted the require relative completion a bit to account for this possibility

### Automated Tests

Added a test showing the behaviour.

### Manual Tests

1. Start the LSP on this branch
2. Type `require "` or `require_relative "` into any file
3. Verify that you get completions offered right away
